gilbreth_perception/src/segmentation_node.cpp

```diff
@@ -19,12 +33,68 @@ uint32_t g_min_cluster_size = 80;
 uint32_t g_max_cluster_size = 25000;
 double g_cluster_tolerance = 0.1;
 
-ros::Publisher pub;
+ros::Publisher pub;   //publish PCD
+ros::Publisher pub_v; //publish voxel
```

Comment:
Given the significant changes made to this node I think it'd be best to make a new voxelizer_node.  It would essentially follow the same logical steps but it would only publish the voxel grid message

Reply:
Modified.